CONTRIBUTING.rst

```diff
@@ -0,0 +1,230 @@
+Contributing to dirty_cat
+=========================
+
+First off, thanks for taking the time to contribute!
+
+The following is a set of guidelines for contributing to
+`dirty_cat <https://github.com/dirty-cat/dirty_cat>`__.
+
+.. raw:: html
+
+   <hr />
+
+Table of contents
+-----------------
+
+`I don’t want to read the whole thing I just have a
+question! <#i-dont-want-to-read-the-whole-thing-i-just-have-a-question>`__
+
+`What should I know before I get
+started? <#what-should-i-know-before-i-get-started>`__
+
+`How can I contribute? <#how-can-i-contribute>`__ - `Reporting
+bugs <#reporting-bugs>`__ - `Suggesting
+enhancements <#suggesting-enhancements>`__ - `Writing your first Pull
+Request <#writing-your-first-pull-request>`__
+
+I don’t want to read the whole thing I just have a question!
+------------------------------------------------------------
+
+We use GitHub Discussions for general chat and Q&As. `Check it
+out! <https://github.com/dirty-cat/dirty_cat/discussions>`__
+
+What should I know before I get started?
+----------------------------------------
+
+If you want to truly understand what are the incentives behind
+dirty_cat, and if scientific literature doesn’t scare you, we greatly
+encourage you to read the two papers `Similarity encoding for learning
+with dirty categorical variables <https://hal.inria.fr/hal-01806175>`__
+and `Encoding high-cardinality string categorical
+variables <https://hal.inria.fr/hal-02171256v4>`__.
+
+How can I contribute?
+---------------------
+
+Reporting bugs
+~~~~~~~~~~~~~~
+
+Even if we unit-test our code, using the library is the best way to
+discover new bugs and limitations.
+
+If you stumble upon one, please - `Check if a similar or identical issue
+already
+exists <https://github.com/dirty-cat/dirty_cat/issues?q=is%3Aissue>`__ -
+If yes: - **The issue is still open**: leave the emote :+1: on the
+original message, which will let us know there are several users
+affected by this issue - **The issue has been closed**: - **It has been
+closed by a merged pull request** (1) update your dirty_cat version, or
+(2) the fix has not been released in a version yet - **Otherwise**,
+there might be a ``wontfix`` label, and / or a reason at the bottom of
+the conversation - If not - `File a new
+issue <https://github.com/dirty-cat/dirty_cat/issues/new>`__ (see
+following section)
```

Comment:
These should be bullet points
```suggestion
If you stumble upon one, please `check if a similar or identical issue already
exists <https://github.com/dirty-cat/dirty_cat/issues?q=is%3Aissue>`__
- If yes: 
  - **The issue is still open**: leave the emote :+1: on the original message, 
    which will let us know there are several users affected by this issue 
  - **The issue has been closed**: 
    - **It has been closed by a merged pull request** (1) update your dirty_cat version, 
      or (2) the fix has not been released in a version yet
    - **Otherwise**, there might be a ``wontfix`` label, and / or a reason at the bottom of the conversation 
- If not, `file a new issue <https://github.com/dirty-cat/dirty_cat/issues/new>`__ (see following section)
```